docs/setup.md

```diff
@@ -2,45 +2,64 @@
 
 This document outlines the setup process for the `Superposition Platform`.
 
-## Prerequisites
+## Installation for linux/MacOS
 
-* Docker installed and running.
-* Docker Compose installed.
-* [Follow installation instructions at Zero To Nix ↗](https://zero-to-nix.com/start/install) 
+- Install Rust using rustup
+    `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+- Install wasm-pack
+    `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
+- Install docker
+- make sure you have dependencies like libpq, openssl, libiconv,etc installed
+- make sure `make` command is installed
+  - Linux
+    ```bash
+    sudo apt-get install build-essential
+    ```
+  - MacOS
+    ```bash
+    xcode-select --install
+    ```
+- Clone the repository 
+    ```bash
+    git clone https://github.com/juspay/superposition.git
+    cd superposition
+    ```
+- Start docker daemon
+    - MacOS
```

Comment:
better to keep the order the same in all examples (Linux first and then MacOS for consistency)